robusta_krr/core/runner.py

```diff
@@ -298,7 +306,11 @@ async def run(self) -> None:
             result = await self._collect_result()
             logger.info("Result collected, displaying...")
             self._process_result(result)
-        except ClusterNotSpecifiedException as e:
-            logger.error(e)
+        except (ClusterNotSpecifiedException, CriticalRunnerException) as e:
```

Comment:
maybe not for now but you can put specific errors in the pod status this way
https://kubernetes.io/docs/tasks/debug/debug-application/determine-reason-pod-failure/#customizing-the-termination-message
We might want to do that in the future as we get more types of exceptions